frontend/src/components/common/NewTable/Table.styled.ts

```diff
@@ -156,7 +156,7 @@ export const Table = styled.table(
     & a {
       color: ${table.td.color.normal};
       font-weight: 500;
-      max-width: 450px;
+      max-width: 150ch;
```

Comment:
as far as I can see we do not use `ch` anywhere yet and not sure how introducing these is legit. @Mgrdich ideas?